### PR TITLE
pod: fix segfault when there are no arguments to inspect

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/containers/libpod/issues/2681

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@mheon @baude @TomSweeneyRedHat @giuseppe @jwhonce PTAL

**Comment by rhatdan** [comment]
@tobwin PTAL

File: cmd/podman/pod_inspect.go [hunk 1]
```diff
@@ -44,6 +44,11 @@ func podInspectCmd(c *cliconfig.PodInspectValues) error {
 		pod *adapter.Pod
```
Comment (TomSweeneyRedHat) [inline cmd/podman/pod_inspect.go]: I can't comment up higher, but the message at line 37 says:  "Act on the latest container podman is aware of".  I think that should be "latest pod podman".

**Comment by rhatdan** [comment]
/lgtm
I think we should merge this as is and open a second PR to fix those Cut and Paste errors.

**Pull request merged**